bin/pa11y.js

```diff
@@ -298,22 +300,14 @@ function collectOptions(val, array) {
 
 /**
  * Output environment info for debugging purposes
- * @param {Object} pkg - The package.json object
- * @returns {void}
+ * @returns {Promise} - resolves a string with environment information from envinfo
  */
 function outputEnvironmentInfo() {
-	const versions = {
-		pa11y: pkg.version,
-		node: process.version.replace('v', ''),
-		npm: '[unavailable]',
-		os: require('os').release()
-	};
-	try {
-		versions.npm = require('child_process').execSync('npm -v').toString().trim();
-	} catch (error) {}
-
-	console.log(`Pa11y:      ${versions.pa11y}`);
-	console.log(`Node.js:    ${versions.node}`);
-	console.log(`npm:        ${versions.npm}`);
-	console.log(`OS:         ${versions.os} (${process.platform})`);
+	return envinfo.run({
+		System: ['OS', 'CPU', 'Memory'],
```

Comment:
`'Shell'` would probably be useful.

```suggestion
		System: ['OS', 'CPU', 'Memory', 'Shell'],
```